### ISSUE TITLE
Add audio journaling feature to everyday journals feature

### ISSUE THREAD

**Issue by Anushka-Pote** [Amna-Hassan04/Serenity-Guide#83]
The journaling feature is awesome for users to input their emotions and thoughts into the application, but typing all the emotions is quite time consuming. Thus, adding an audio input for journaling feature makes it easy, quick and efficient for the users

**Comment by Amna-Hassan04** [comment]
@Anushka-Pote Yeah, you can work on this, make sure the new feature looks according to the games aesthetic and you can share your progress here with screenshots and ping me if you need any help.

**Comment by Anushka-Pote** [comment]
![image](https://github.com/user-attachments/assets/fb62c1e2-273a-4f3e-82d9-af8bd7d32b71)

Hey, 
I am working on it and basic work has been done. Just have a look at it once though a lot of works has been pending as adding the backend storage for audio is quite a task.
